src/homeWork6/Car.java

```diff
@@ -0,0 +1,22 @@
+package homeWork6;
+
+public class Car {
+
```

Comment:
Would be better to organize methods in the following way: public methods first and private methods in the end.